packages/google-pay-integration/src/gateways/google-pay-adyenv3-gateway.ts

```diff
@@ -0,0 +1,121 @@
+import {
+    AdyenAction,
+    AdyenAdditionalAction,
+    AdyenAdditionalActionState,
+    AdyenClient,
+    AdyenError,
+    AdyenV3ScriptLoader,
+    isAdditionalActionRequiredErrorResponse,
+} from '@bigcommerce/checkout-sdk/adyen-utils';
+import {
+    getBrowserInfo,
+    isRequestError,
+    MissingDataError,
+    MissingDataErrorType,
+    NotInitializedError,
+    NotInitializedErrorType,
+    Payment,
+    PaymentIntegrationSelectors,
+    PaymentIntegrationService,
+    PaymentMethod,
+    PaymentMethodCancelledError,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+// import { assertsIsGooglePayAdyenV3InitializationData } from '../guards/is-google-pay-adyen-initialization-data';
```

Comment:
Do you still need this comment?

Reply:
If not, I think, its ok to remove `assertsIsGooglePayAdyenV3InitializationData` method from code base